api/v1beta1/awscluster_conversion.go

```diff
@@ -56,6 +56,7 @@ func (src *AWSCluster) ConvertTo(dstRaw conversion.Hub) error {
 	if restored.Status.Bastion != nil {
 		dst.Status.Bastion.InstanceMetadataOptions = restored.Status.Bastion.InstanceMetadataOptions
 		dst.Status.Bastion.PlacementGroupName = restored.Status.Bastion.PlacementGroupName
+		dst.Status.Bastion.PlacementGroupPartition = restored.Status.Bastion.PlacementGroupPartition
```

Comment:
No need to add this new filed in v1beta1 as we would be going to deprecate that soon

Reply:
Okay, that makes sense. I'm wondering if CI tests would complain if I remove the conversion. If that's not the case, then we can remove it.

Reply:
The test failed after I removed the conversion. Can you please suggest what should be the correct approach here?

```
--- FAIL: TestFuzzyConversion (59.20s)
    --- FAIL: TestFuzzyConversion/for_AWSCluster (18.80s)
        --- PASS: TestFuzzyConversion/for_AWSCluster/spoke-hub-spoke (18.79s)
        --- FAIL: TestFuzzyConversion/for_AWSCluster/hub-spoke-hub (0.01s)
    --- FAIL: TestFuzzyConversion/for_AWSMachine (17.20s)
```

Reply:
Couldn't we just leave this generated conversion in place if it is causing issues? I guess if we deprecate v1beta1, it won't matter either way

Reply:
I think that would be better, since it's causing the CI tests failure. I've added back the conversion.